schemas/github-issue-forms.json

```diff
@@ -0,0 +1,2377 @@
+{
```

Comment:
Is it necessary to add this to the repo? Like is there a way to do this remotely?

Reply:
I don't think there's another way